python/AMI_NOT_PUBLIC_CHECK/AMI_NOT_PUBLIC_CHECK_test.py

```diff
@@ -0,0 +1,211 @@
+import sys
+import unittest
+try:
+    from unittest.mock import MagicMock, patch, ANY
+except ImportError:
+    import mock
+    from mock import MagicMock, patch, ANY
+import botocore
+from botocore.exceptions import ClientError
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::::Account'
+
+#############
+# Main Code #
+#############
+
+config_client_mock = MagicMock()
+sts_client_mock = MagicMock()
+ec2_client_mock = MagicMock()
+
+class Boto3Mock():
+    def client(self, client_name, *args, **kwargs):
+        if client_name == 'config':
+            return config_client_mock
+        elif client_name == 'sts':
+            return sts_client_mock
+        elif client_name == 'ec2':
+            return ec2_client_mock
+        else:
+            raise Exception("Attempting to create an unknown client")
+
+sys.modules['boto3'] = Boto3Mock()
+
+rule = __import__('AMI_NOT_PUBLIC_CHECK')
+
+# Checks for scenario wherein no non-compliant resources are present
+class CompliantResourcesTest(unittest.TestCase):
+    lambda_event = {}
+
+    def setUp(self):
+        self.lambda_event = build_lambda_scheduled_event()
+        pass
+
+    def test_compliant_resources(self):
+        describe_images_result = {'Images': [],
+                                  'ResponseMetadata': {'RequestId': 'c9e93cf7-7503-4d7a-a012-4b9f0461de31',
+                                                       'HTTPStatusCode': 200,
+                                                       'HTTPHeaders': {'content-type': 'text/xml;charset=UTF-8',
+                                                                       'content-length': '219',
+                                                                       'date': 'Thu, 21 Mar 2019 10:33:35 GMT',
+                                                                       'server': 'AmazonEC2'},
+                                                       'RetryAttempts': 0}}
+        ec2_client_mock.describe_images = MagicMock(return_value=describe_images_result)
+        lambda_result = rule.lambda_handler(self.lambda_event, {})
+        expected_response = [build_expected_response(compliance_type='NOT_APPLICABLE',
+                                                     compliance_resource_id='N/A',
+                                                     compliance_resource_type=DEFAULT_RESOURCE_TYPE)]
```

Comment:
These indentation can be better, like below:
```python
expected_response = [
		build_expected_response(
			compliance_type='NOT_APPLICABLE',
			compliance_resource_id='N/A',
			compliance_resource_type=DEFAULT_RESOURCE_TYPE
		)
]
```

Reply:
Done